src/sentry/migrations/0268_rename_issue_widget_query_fields.py

```diff
@@ -0,0 +1,57 @@
+# Generated by Django 2.2.24 on 2021-09-23 14:02
+from django.db import migrations
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def rename_issue_widget_query_fields(apps, schema_editor):
+
+    DashboardWidgetQuery = apps.get_model("sentry", "DashboardWidgetQuery")
+
+    old_to_new_field_mapping = {
+        "count": "events",
+        "userCount": "users",
+        "lifetimeCount": "lifetimeEvents",
+        "lifetimeUserCount": "lifetimeUsers",
```

Comment:
Could users have queries for custom tags that overlap with these names?

Reply:
Hmm good question, theres a possibility but I think this concern would also be true for any of our other fields. Do we have any safeguards for this? I'll see if I can do some digging to see if anyone is using these as tags (don't think I'll be able to do this for self hosted)

Reply:
Hey Mark, on second thought, there shouldn't be any concern if there are overlaps since these fields are for Issue Widgets only and we don't allow selecting tags on columns for Issue Widgets right now ie there wouldn't be any collisions because Issue column fields are a fixed set. Let me know if you have any thoughts around this!

Reply:
Sounds good to me. I didn't know if we allowed users to select arbitrary fields in issue queries.